src/instance_info.rs

```diff
@@ -153,10 +154,10 @@ impl InstanceInfo {
                 Deploy date: {}\n
                 Deploy timestamp: {}\n
                 Compile mode: {}\n
+				SFW only: {:?}\n
```

Comment:
I agree we should move this around, but I think we should wait to do this in another deliverable.

Reply:
Ah, thanks. I should have only added the diffs that were relevant but I rather removed the diffs that looked irrelevant.